HeroArena/src/main/java/com/revature/controller/LoginController.java

```diff
@@ -4,13 +4,20 @@
 
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.*;
+<<<<<<< HEAD
+=======
```

Comment:
Why is this here? This seems to be part of a merge conflict that wasn't properly resolved?

Reply:
Sorry, removed it.

Reply:
Please commit again.

Reply:
it's been pushed

Reply:
ok, dont know why it went back